src/test/ui/imports/reexports.rs

```diff
@@ -1,16 +1,22 @@
+#![warn(unused)]
+
+// ignore-tidy-linelength
```

Comment:
This is better avoided, the `//~ ERROR` or `//~ WARNING` annotations can be placed on the line below the error if necessary.
```rust
pub use super::*;
//~^ WARNING must import ...
```

Reply:
I have made the changes locally but the I am not able to run the test, I'll be able on sunday and I'll commit them after

Reply:
The message after `//~^ WARNING` doesn't necessary need to be the full output, any prefix like `//~^ WARNING this glob doesn't reexport anything` will match too.
Using a prefix is preferable to `// ignore-tidy-linelength`, full messages are tested by the `.stderr` files anyway.